.ruff.toml

```diff
@@ -11,4 +11,5 @@ ignore = [
 	"E501", # line too long
 ]
 [per-file-ignores]
-"__init__.py" = ["F401"]
\ No newline at end of file
+"__init__.py" = ["F401"]
+"variableForUserParams_pb2.py" = ["F821", "N999", "E712"]
```

Comment:
I think this might support wildcards - maybe we can do something like `*_pb2.py`?